### DISCUSSION TITLE
Seberapa pentingkah melakukan tracing referensi?

### DISCUSSION THREAD

**Discussion by taruma** [hidrokit/hidrokit#154] [Umum]
Diskusi: Seberapa pentingkah melakukan _tracing_ referensi?

_Tracing_ yang dimaksud adalah melakukan lacak balik teori yang digunakan di hidrokit dalam analisis hidrologi. Sejauh ini saya menemukan beberapa referensi yang sulit dilacak balik seperti dulu saya mengalami kesulitan untuk melihat makalah asli untuk pemodelan FJ Mock. Dan sekarang saya juga mengalami kesulitan lacak balik untuk teori uji abnormalitas, tabel jenis sebaran (Ck, Cs).

Saat ini sudah ada perhitungan dalam bentuk excel terkait referensi tersebut, dan saya bisa gunakan referensi tersebut tanpa pertanyaan dan saya bisa membela diri berdasarkan keseringannya referensi tersebut di laporan yang telah digunakan. Tapi, saya masih ada unsur keraguan apakah itu langkah yang tepat atau tidak. Ini juga terkait waktu yang dibutuhkan untuk melacak balik, yang sebenarnya bisa dialihkan ke pengembangan ataupun pekerjaan lainnya yang lebih signifikan. 

Pendapat saya sih cukup penting untuk memastikan teori dan penerapan/pengembangan kodenya konsisten. Saya mengasumsikan bisa terjadi _lost in translation_ setelah perhitungan/informasi diulang di berbagai laporan tanpa menjelaskannya secara mendetail. 

Kembali ke dalam bentuk pertanyaan, seberapa pentingkah kita melakukan _tracing_ untuk teori/referensi/perhitungan yang kita gunakan dalam analisis?

**Comment by iingLK** [comment]
Saya rasa lebih amannya untuk tidak perlu melakukan tracing, agar agile manifesto bisa dipenuhi. Karena jika tracing dilakukan akan time consuming, dan akan deviate dari tujuan agile manifesto.

Pun, apabila ingin diterapkan tracing. Perlu dinotice kepada semua yang berkepentingan bahwa akan ada alur penggunaan sumberdaya yang mungkin tidak direncanakan atau tidak disadari sebelumnya.

**Reply by taruma** [reply]
Mengingat pada saat ini waktu yang dibutuhkan untuk tracing cukup lama, solusi saya untuk sementara menjadikan hal tersebut dalam bentuk diskusi atau tanya-jawab, dan jika ada waktu untuk tracing baru deh dilakukan. 

Terima kasih atas sarannya @iingLK.
